Wireframe/index.html

```diff
@@ -4,9 +4,57 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
+    <article>
+    <section>
+    <header>
+      <h1 class="github-header-title">Github</h1>
+      <h4>Article surrounding Github questions</h4>
+    </header>
+    <main>
+      <div class="what-is-git-container">
+      <img src="https://static.vecteezy.com/system/resources/previews/017/119/660/original/github-logo-git-hub-icon-with-text-on-white-and-black-background-free-vector.jpg" class="what-is-git-img"/>
+      <div class="title-desc-btn">
+      <h1>What is Git?</h1>
+      <h4>Distributed version control for efficient code collaboration.</h4>
+      <a href="#">
+      <button class="read-more-btn">Read More</button>
+      </a>
+      </div>
+      </div>
+    </main>
+    </section>
+
+    <section>
+        <div class="why-do-devs-and-branch-container">
+        <img
+          src="https://www.cyberark.com/wp-content/uploads/2019/11/Developer.jpg"
+          class="why-do-devs-img" />
+        <div class="-title-desc-btn">
+          <h1>Why do developers need Git?</h1>
+          <h4>Empowers developers with efficient collaboration and version control.</h4>
+          <a href="#">
```

Comment:
I think a link goes somewhere and a button triggers an action. Pick one or the other. Which one would you pick?